README.md

```diff
@@ -32,24 +32,24 @@ s puppies
 
 Search for a wifi router on amazon
 ```
-s -p amazon wifi router
+s wifi router -p amazon
```

Comment:
You should actually change `wifi router` to `"wifi router"` just to make sure there won't be any issues with using a dash.